packages/core/src/web/helpers/device/framing.ts

```diff
@@ -450,6 +506,26 @@ class FramingTaskManager extends EventEmitter {
       return res;
     }
 
+    if (type === FramingType.RotateAxis) {
+      if (this.rotaryInfo?.y === undefined) {
+        return [];
+      }
+
+      const { width } = workareaManager;
+
+      const res: Array<[number, number]> = [
+        [0, this.rotaryInfo.y],
+        [width / constant.dpmm, this.rotaryInfo.y],
+        [width / constant.dpmm, this.rotaryInfo.y],
+        [0, this.rotaryInfo.y],
+        [0, this.rotaryInfo.y],
+      ];
```

Comment:
Is the duplicated `[width / constant.dpmm, this.rotaryInfo.y]` and `[0, this.rotaryInfo.y]` added on purpose?